src/bootstrap/doc.rs

```diff
@@ -834,6 +834,7 @@ macro_rules! tool_doc {
                 cargo.arg("-Zskip-rustdoc-fingerprint");
                 // Only include compiler crates, no dependencies of those, such as `libc`.
                 cargo.arg("--no-deps");
+                cargo.arg("--lib");
```

Comment:
this is passed for all tools, not just bootstrap. was that intentional?

Reply:
oops, missed it. Will fix it

Reply:
I'm sorry, I made a mistake. Thank you for your correction.